WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/TilePreviewActivity.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.wear.tiles
+
+import android.content.ComponentName
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.wear.tiles.manager.TileUiClient
+
+
+/**
+ * Previews a Tile in an activity, using the TileUiClient from the wear-tiles-renderer library.
+ */
+class TilePreviewActivity : ComponentActivity(R.layout.activity_tile_preview) {
```

Comment:
I'm curious if you need a separate activity for each preview.  Are you trying to show these as standalone activities as well?

Is it worth trying a wrapper composable with the TilesUiClient inside an AndroidView?

It would keep us following the Single Activity Architecture here.

Your call.

Reply:
>Are you trying to show these as standalone activities as well?

No, just wanted to quickly separate a list and detail view.

Happy to follow up on this after I go through the Compose nav docs; having it as a single activity will be a good improvement.